examples/peers.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example nodeinfo --release
+use iota::Client;
+
+/// In this example we get the nodeinfo
+#[tokio::main]
+async fn main() {
+    let iota = Client::builder() // Crate a client instance builder
+        .with_network("alphanet1")
```

Comment:
Please remove this, it's not needed and could cause problems

Reply:
you mean only line 11, correct?

Reply:
Yes